integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java

```diff
@@ -66,15 +65,15 @@
 import static oracle.weblogic.kubernetes.actions.TestActions.patchDomainCustomResource;
 import static oracle.weblogic.kubernetes.assertions.TestAssertions.secretExists;
 import static oracle.weblogic.kubernetes.assertions.TestAssertions.verifyRollingRestartOccurred;
+import static oracle.weblogic.kubernetes.assertions.impl.Domain.doesDomainExist;
```

Comment:
Move the method into standard package naming convention such as 

oracle.weblogic.kubernetes.assertions.impl.Domain.doesDomainExist -->
oracle.weblogic.kubernetes.assertions.TestAssertions.doesDomainExist

Reply:
Actually there is one in TestAssertions already :)
public static Callable<Boolean> domainDoesNotExist(String domainUid, String domainVersion, String namespace) {
    return () -> !Domain.doesDomainExist(domainUid, domainVersion, namespace);
  }
Ok, let me call that one instead

Reply:
@anpanigr I modified the code to use TestAssertion.doesDomainExist and rerun the single ItMiiAuxiliary test.
OKD:
https://build.weblogick8s.org:8443/job/wko-okd-test/73/testReport/oracle.weblogic.kubernetes/ItMiiAuxiliaryImage/
kind new
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/7222/testReport/oracle.weblogic.kubernetes/ItMiiAuxiliaryImage/